fdbserver/TagPartitionedLogSystem.actor.cpp

```diff
@@ -2423,6 +2424,25 @@ ACTOR Future<Void> TagPartitionedLogSystem::epochEnd(Reference<AsyncVar<Referenc
 			logSystem->stopped = true;
 			logSystem->pseudoLocalities = prevState.pseudoLocalities;
 
+			if (SERVER_KNOBS->ENABLE_VERSION_VECTOR_TLOG_UNICAST) {
+				logSystem->recoverAt = getRecoverVersionUnicast(logGroupResults, minEnd);
+				TraceEvent("RecoveryVersionInfo").detail("RecoverAt", logSystem->recoverAt);
+				// When a new log system is created, inform the surviving tLogs of the RV.
+				// SOMEDAY: Assert surviving tLogs use the RV from the latest log system.
+				for (auto logGroupResult : logGroupResults) {
```

Comment:
Thanks for documenting this "todo" item! As I mentioned to you, that can be done later and we don't need to block this PR for that.